NORSE_actor/hdf5Write.py

```diff
@@ -0,0 +1,60 @@
+# hdf5Write.py
+# 12/12/2019
+# Written by Soma Olasz
+#
+# This function is created to write data to hdf5 file.
+
+import h5py
+import os
+import pwd
+
+
+def write(shotnumber, runnumber, *args):
```

Comment:
Would be safer from the point of view of data consistency to used a dictionary to pass data and respective names.

Reply:
Done.